### ISSUE TITLE
ChangedBy value cannot be empty when BypassRules is specified

### ISSUE THREAD

**Issue by pedrolbarroso** [solidify/jira-azuredevops-migrator#161]
Hello. I'm getting the following error running the migration:
ChangedBy value cannot be empty when BypassRules is specified

I appreciate any help!

**Comment by madkoo** [comment]
@pedrolbarroso can you provide the import log file and the item json that fail? Then we can investigate where the problem might be.

**Comment by madkoo** [comment]
@pedrolbarroso are you still experiencing the problem described? Otherwise we will soon close this issue.

**Comment by MOlausson** [comment]
Closing in lack of further response. @pedrolbarroso feel free to re-open if you still are having problems.

**Comment by jackpoz** [comment]
This issue happened to me today too using another software but with the exact same error. While troubleshooting I changed WorkItemStoreFlags from BypassRules to None and received a different error about the work item type being disabled. I checked in the process, re-enabled it and that fixed the "ChangedBy value cannot be empty when BypassRules is specified" issue.

Hope this will help people reaching this github issue when googling 😃
